spec/customer_store/attribute_definition_endpoints_spec.rb

```diff
@@ -45,22 +45,22 @@
 
       it 'should return the status' do
         expect do
-          expect(client.get_attribute_definitions('the-oid')['status'])
+          expect(client.get_attribute_definitions('the-oid').status)
             .to eq(500)
-        end.to call_customer_store.get_attribute_definitions('the-oid',
-                                                             'status' => 500)
+        end.to call_customer_store
+          .get_attribute_definitions('the-oid', 'status' => 500)
       end
 
       it 'should return the headers' do
         expect do
-          expect(client.get_attribute_definitions('the-oid')['headers'])
+          expect(client.get_attribute_definitions('the-oid').headers)
```

Comment:
You can still use treat the response object as if it's a hash, and I left it this way in stubs and mock files in `/lib/shore/customers/rspec`